bot.py

```diff
@@ -470,18 +476,20 @@ async def notify_if_voice_client_is_missing(self, message):
             return True
         return False
 
-    async def play(self, message, command_content):
+    async def play(self, message, command_content, deque_append_left=False):
         """
         Play URL or first search term from command_content in the author's voice channel
         """
+        # pylint: disable=too-many-branches
+
```

Comment:
This is a warning which shouldn't be ignored IMO.

Rather than ignoring it I would prefer to move some of the logic to their own function, e.g. `def invalid_play` or combine some of the `not command_content` branches.

Some branches could be moved to `def get_media_data(term)`

Reply:
done